[website] Open "React Engineer - Material UI Maintainer" role (#45937)
mui/material-ui: docs/pages/careers/react-community-engineer-core.md

We also host Design Kits and pre-built Templates.

Beyond the core libraries, MUI X offers advanced components like the Data Grid, Date and Time Pickers, and Charts, for more complex user interactions and data visualization needs.

Learn more about MUI's products in this blog post: [An introduction to the MUI ecosystem](https://mui.com/blog/mui-product-comparison/).

## The culture

MUI is a fully remote company with a team that spans across the globe.
The majority of our work is asynchronous, and we rely mostly on written communication to collaborate.
We're radically transparent: nearly all of our work happens in public.
Each contributor has the freedom to decide how and when they work, and that work is primarily self-directed: it's your responsibility to define and complete your own tasks in a timely manner.

For additional details about the culture, you can check our [careers](https://mui.com/careers/) and [about](https://mui.com/about/) pages and also our [public Handbook](https://mui-org.notion.site/Handbook-f086d47e10794d5e839aef9dc67f324b).

## Why we're hiring

We need a Community Engineer to solve friction points of users.
We face challenges on multiple fronts:

> - As a result of this growth, we (will) have more users questions and feedback coming in than ever before.

Reviewer: This is a bit confusing. What "growth" are you referring to?

Author: ```suggestion
- As the library grows, we (will) have more users questions and feedback coming in than ever before.
```

Is it more clear?

- The rapid growth of new products is stretching our developers too thin.
- Software engineers at MUI are tasked with doing support, however, they could benefit from the help of a dedicated person per project, for instance to improve the workflows and tooling.

Overall, both our open-source community and our premium products are growing fast.
We need talented people to keep up with the demands of our exigent community!

## The role

### Why this is interesting

> You will be a key member of the Developer Experience's team and will directly impact customer satisfaction and success.

Reviewer: To double check, should it be "core team"?
It might confuse the joiner because we do have Developer Experience team.

Author: ```suggestion
You will be a key member of the Core team and will directly impact the project's roadmap.
```

Our products empower React developers to build awesome applications faster – we see millions of developers on MUI's docs every year, one million a month.

### Key responsibilities

You will own the following responsibilities:

- **Build product knowledge**. You will continually research and learn the current and future best practices of using MUI.
- **Resolve users' issues**. You will solve these issues at two levels:
  - On the surface, you will answer developers on GitHub.
  - At the root, you will create/update the documentation, fix bugs, implement community driven features and more.
- **Provide feedback**. You will work alongside product managers to define and shape the product goals, roadmap, priorities, and strategy based on your frontline knowledge of customer needs.
- **Operations**.
  - You will establish key support metrics and identify how best to measure them.
  - You will establish a workflow to reduce 'time to response' and 'time to fix' that can scale to multiple team members.
  - You will identify where internal tooling might be developed or obtained to improve support efficiency.

Depending on the day, you'll:

> - **You'll be interacting with the users** on a regular basis, handling inbound support and feature requests (every developer helps with developer requests).

Reviewer: ```suggestion
- **Interface with the users** on a regular basis, handling inbound support and feature requests (every developer helps with developer requests).
```

  It would be great if you could address this in your cover letter!
- **You've contributed to a design system before**.
  You have worked on enterprise components in the past, maybe for internal use in your company.
  You have built complex and advanced features while making sure the accessibility was fully functional.

### Nice to have (but not required)

- **You've maintained an active repository before**.
  Maybe you've helped maintain a popular open-source repository, or perhaps you've worked on internal repositories that saw contributions from multiple teams.
  Previous experience with highly active repository workflows is a definite plus for this role.
- **You have contributed code**. A history of contributing to MUI's open-source projects is a plus.

## Benefits and compensation

Competitive compensation depending on the profile and location.
We are ready to pay top market rates for a person who can clearly exceed the role's expectations.
You can find the other perks & benefits on the [careers](https://mui.com/careers/#perks-and-benefits) page.

## How to apply

> [Apply now for this position 📮](https://jobs.ashbyhq.com/MUI/__ID__/application?utm_source=ZNRrPGBkqO)

Author: ```suggestion
[Apply now for this position 📮](https://jobs.ashbyhq.com/MUI/f28a1482-22fb-4cf8-9263-092ec0f81657/application?utm_source=ZNRrPGBkqO)
```
